x-pack/plugin/identity-provider/src/test/java/org/elasticsearch/xpack/idp/saml/sp/SamlServiceProviderDocumentTests.java

```diff
@@ -145,6 +146,7 @@ private SamlServiceProviderDocument assertXContentRoundTrip(SamlServiceProviderD
 
             final BytesReference bytes2 = XContentHelper.toXContent(obj2, xContentType, humanReadable);
             assertThat(bytes2, equalTo(bytes1));
+            assertToXContentEquivalent(bytes1, bytes2, xContentType);
```

Comment:
For me, it was the line before that failed, so the test wouldn't reach this new assertion. Am I missing something?

Reply:
merge woes